pytermgui/cmd.py

```diff
@@ -10,35 +10,20 @@
 from __future__ import annotations
 
 import os
-import sys
 import platform
 import subprocess
-from random import randint
-from itertools import zip_longest
+import sys
 from abc import ABC, abstractmethod
-from typing import Any, Optional, Type
 from argparse import ArgumentParser, Namespace
+from itertools import zip_longest
+from random import randint
+from typing import Any, Optional, Type
 
-from . import (
-    __version__,
-    MarkupSyntaxError,
-    # prettify_markup,
-    MarkupFormatter,
-    WindowManager,
-    real_length,
-    YamlLoader,
-    get_widget,
-    InputField,
-    Container,
-    Splitter,
-    terminal,
-    markup,
-    Window,
-    Button,
-    Label,
-    boxes,
-    keys,
-)
+from . import Label  # prettify_markup,
+from . import (Button, Container, InputField, MarkupFormatter,
+               MarkupSyntaxError, Splitter, Window, WindowManager, YamlLoader,
+               __version__, boxes, get_widget, keys, markup, real_length,
+               terminal)
```

Comment:
I don't agree with these changes, `black` will modify it as well.

I would also argue it is a bit odd to make formatting changes to all files in a library, even ones you didn't manually touch, but that's kind of on the formatter.